language-server/modules/langserver-core/src/test/java/org/ballerinalang/langserver/completion/definitions/FunctionDefinitionCompletionTest.java

```diff
@@ -43,8 +43,8 @@ public void test(String config, String configPath) throws WorkspaceDocumentExcep
     public Object[][] dataProvider() {
         log.info("Test textDocument/completion for Function Definition Scope");
         return new Object[][] {
-                {"emptyLineCompletion.json", "function"},
-                {"nonEmptyLineCompletion.json", "function"},
+//                {"emptyLineCompletion.json", "function"},
```

Comment:
Any reason why we have disable these tests?

Reply:
Because completion tests are failing. I have reported an issue, the team is fixing them.